### PR TITLE
Add user change password form

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

**Comment by itkg-rboessel** [comment]
open-orchestra/open-orchestra#475
open-orchestra/open-orchestra-user-bundle#51
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/777

File: UserBundle/Form/Type/ChangePasswordUserType.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace OpenOrchestra\UserBundle\Form\Type;
+
+use Symfony\Component\Form\FormBuilderInterface;
+use FOS\UserBundle\Form\Type\ChangePasswordFormType;
+use OpenOrchestra\BackofficeBundle\EventSubscriber\AddSubmitButtonSubscriber;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ChangePasswordUserType extends ChangePasswordFormType
```
Comment (nicolasThal) [inline UserBundle/Form/Type/ChangePasswordUserType.php]: class comment

File: UserBundle/Form/Type/ChangePasswordUserType.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    /**
+     * @param string $class The User class name
+     */
+    public function __construct($class)
+    {
+        $this->class = $class;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
```
Comment (nicolasThal) [inline UserBundle/Form/Type/ChangePasswordUserType.php]: method comment

File: UserBundle/Form/Type/ChangePasswordUserType.php [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    {
+        parent::buildForm($builder, $options);
+
+        $builder->addEventSubscriber(new AddSubmitButtonSubscriber());
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
```
Comment (nicolasThal) [inline UserBundle/Form/Type/ChangePasswordUserType.php]: method comment

File: UserBundle/Form/Type/ChangePasswordUserType.php [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    {
+        $resolver->setDefaults(array(
+            'data_class' => $this->class,
+            'intention'  => 'change_password',
+        ));
+    }
+
+    public function getName()
```
Comment (nicolasThal) [inline UserBundle/Form/Type/ChangePasswordUserType.php]: method comment

**Pull request closed without merging**